apps/docs/docs/operations/calendar/project-calendar.md

```
@@ -0,0 +1,324 @@
+3. Add Calendar → Create new blank calendar
+4. Name: "Phoenix Rooivalk - Opportunities"
+5. Share with team members
+```
+
+### ICS File Download
+
+Download calendar events as ICS files for import:
+
+| Calendar                  | Download               |
+| ------------------------- | ---------------------- |
+| Opportunities & Deadlines | [opportunities.ics](#) |
```

markdownlint-cli2 (0.18.1)

No empty links

(MD042, no-empty-links)